shell/platform/darwin/ios/framework/Headers/FlutterEngineGroup.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+
+#import "FlutterEngine.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Represents a collection of FlutterEngines who share resources which allows
+ * them to be created with less time const and occupy less memory than just
+ * creating multiple FlutterEngines.
+ *
+ * Deleting a FlutterEngineGroup doesn't invalidate existing FlutterEngines, but
+ * it eliminates the possibility to create more FlutterEngines in that group.
+ *
+ * @see http://flutter.dev/go/multiple-engines
+ */
+@interface FlutterEngineGroup : NSObject
+- (instancetype)init NS_UNAVAILABLE;
+
+/**
+ * Initialize a new FlutterEngineGroup.
+ *
+ * @param name The name that will present in the threads shared across the
+ * engines in this group.
+ * @param project The `FlutterDartProject` that all FlutterEngines in this group
+ * will be executing.
+ */
+- (instancetype)initWithName:(NSString*)name
+                     project:(nullable FlutterDartProject*)project NS_DESIGNATED_INITIALIZER;
+
+/**
+ * Creates a running `FlutterEngine` that exists in this group.
```

Comment:
Say that creating the first one and subsequent ones behave differently? (memory and latency wise)

Reply:
Addressed in xiao feedback 4 - That information is in the FlutterEngineGroup.  I reworded it and added a link to the class.